src/Distribution/Client/Mirror/Repo/Secure.hs

```diff
@@ -186,7 +186,7 @@ finalizeLocalMirror' cache repoRoot = (`runContT` return) $ do
     copyFileAtomic src dst = ContT $ \callback -> do
       let (dir, template) = splitFileName dst
       bracket (openBinaryTempFileWithDefaultPermissions dir template)
-              (\(temp, h) -> ignoreIOErrors (hClose h >> removeFile temp)) $
+              (\(temp, h) -> ignoreIOErrors (hClose h >> removeFile temp))
```

Comment:
Here, rather than removing `$`, I would have removed the parentheses around the long term that follows.

Reply:
Makes sense, applied.